experiment.py

```diff
@@ -0,0 +1,119 @@
+import os
+import sys
+import re
+import csv
+import json
+from shutil import copyfile
+from config import Config
+import subprocess
+import os.path
+from os import path
+
+import pandas as pd
+
+def run_experiment(config_file, report_dir):
```

Comment:
Please rename to `run_all_experiments`, then we can refactor the for loop to include a `run_experiment` method. 

Ideally `run_all_experiments` contains only the lines that the experimenter needs to customize - basically the loop start/stop/increment conditions.